shaders/src/light_indirect.fs

```diff
@@ -13,12 +13,31 @@
 
 #define IBL_INTEGRATION_IMPORTANCE_SAMPLING_COUNT   64
 
+//------------------------------------------------------------------------------
+// IBL tinting helpers
+//------------------------------------------------------------------------------
+
+float OverlayBlend(float a, float b) {
+    // Visually almost-luminance-preserving formulation. This uses an artist-provided constant below
+    // that they've found the visually most pleasing under _most_ (but not all) circumstances.
+    const float kT = 0.65;
+    const float kOneOverT = 1.0 / kT;
+    return (a < kT) ? a * kOneOverT * b : a - kT + b;
+}
```

Comment:
Add empty line between functions

Reply:
Fixed in 6dc6210f556cee2f8ce36cfd404588c5ddb3fa2d